flickit-assessment-data/src/main/resources/liquibase/changelog-0.12.xml

```diff
@@ -1004,4 +1004,152 @@
     </changeSet>
 
+    <changeSet id="0.12-179" author="Ali Sedaghat">
+        <renameTable oldTableName="baseinfo_maturitylevel" newTableName="fak_maturity_level"/>
+    </changeSet>
+
+    <changeSet id="0.12-180" author="Ali Sedaghat">
+        <sql>alter table fak_maturity_level rename constraint baseinfo_maturitylevel_pkey to pk_fak_maturity_level;</sql>
+    </changeSet>
+
+    <changeSet id="0.12-181" author="Ali Sedaghat">
+        <sql>alter table fak_maturity_level drop constraint if exists baseinfo_maturitylevel_value_check;</sql>
+    </changeSet>
+
+    <changeSet id="0.12-182" author="Ali Sedaghat">
+        <dropAllForeignKeyConstraints baseTableName="fak_maturity_level"/>
+    </changeSet>
+
+    <changeSet id="0.12-183" author="Ali Sedaghat">
+        <dropUniqueConstraint tableName="fak_maturity_level"
+                              constraintName="baseinfo_maturitylevel_title_profile_id_2682c3b0_uniq"/>
+    </changeSet>
+
+    <changeSet id="0.12-184" author="Ali Sedaghat">
+        <dropUniqueConstraint tableName="fak_maturity_level"
+                              constraintName="baseinfo_maturitylevel_value_profile_id_ddb3a964_uniq"/>
+    </changeSet>
+
+    <changeSet id="0.12-185" author="Ali Sedaghat">
+        <dropUniqueConstraint tableName="fak_maturity_level" constraintName="uq_baseinfo_maturitylevel_code_kit_id"/>
+    </changeSet>
+
+    <changeSet id="0.12-186" author="Ali Sedaghat">
+        <renameColumn tableName="fak_maturity_level" oldColumnName="assessment_kit_id" newColumnName="kit_id"/>
+    </changeSet>
+
+    <changeSet id="0.12-187" author="Ali Sedaghat">
+        <addForeignKeyConstraint baseTableName="fak_maturity_level"
+                                 baseColumnNames="kit_id"
+                                 referencedTableName="fak_assessment_kit"
+                                 referencedColumnNames="id"
+                                 constraintName="fk_fak_maturitylevel_assessmentkit"
+                                 deferrable="true" initiallyDeferred="true"/>
+    </changeSet>
+
+    <changeSet id="0.12-188" author="Ali Sedaghat">
+        <addUniqueConstraint tableName="fak_maturity_level" columnNames="title,kit_id"
+                             constraintName="fak_maturitylevel_title_kit_id"
+                             deferrable="true" initiallyDeferred="true"/>
+    </changeSet>
+
+    <changeSet id="0.12-189" author="Ali Sedaghat">
+        <addUniqueConstraint tableName="fak_maturity_level" columnNames="index,kit_id"
+                             constraintName="fak_maturitylevel_index_kit_id"
+                             deferrable="true" initiallyDeferred="true"/>
+    </changeSet>
+
+    <changeSet id="0.12-190" author="Ali Sedaghat">
+        <addUniqueConstraint tableName="fak_maturity_level" columnNames="code,kit_id"
+                             constraintName="fak_maturitylevel_code_kit_id"/>
```

Comment:
rename to `fak_maturitylevel_code_kit_id`

Reply:
changed to uq_fak_maturitylevel_code_kitid